src/library/waveform.js

```diff
@@ -0,0 +1,57 @@
+// This is the library for creating the waveform
+class Waveform {
+
+  /** This is the constructor for initializing waveform
```

Comment:
nit: Please take a look at the other JS files and try to make consistent. The first line of `/**` comment should not have any text in it.